static/src/javascripts/projects/common/modules/atoms/youtube-player.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+import fastdom from 'fastdom';
+
+import { loadScript } from 'lib/load-script';
+
+const scriptSrc = 'https://www.youtube.com/iframe_api';
+const promise = new Promise(resolve => {
+    if (window.YT && window.YT.Player) {
+        resolve();
+    } else {
+        window.onYouTubeIframeAPIReady = resolve;
+    }
+});
+
+const loadYoutubeJs = () => {
```

Comment:
I know none of these functions are exposed publicly,m but it would be great if we could annotate them with return types (I think for the most part, they are `void`, except `init`)